cmd/zpool/zpool_main.c

```diff
@@ -2393,6 +2426,12 @@ print_status_config(zpool_handle_t *zhp, status_cbdata_t *cb, const char *name,
 		    1 << vs->vs_configured_ashift, 1 << vs->vs_physical_ashift);
 	}
 
+	if (vs->vs_scan_removing != 0) {
+		(void) printf(gettext("  (removing)"));
+	} else if (vs->vs_noalloc != 0) {
+		(void) printf(gettext("  (non-allocating)"));
```

Comment:
I think read-only implies it won't get frees etc, which is different from non-allocating.

I am sure there is a better term here, just read-only is not it.

Reply:
I agree that read-only is not entirely accurate, but it is clear while non-allocating is very accurate, but not very clear. How about: "no writes" or "write locked"?

Reply:
The wording in `zpool status` should be similar to the property name used with `zpool set ...`.

Unfortunately I don't think that "no writes" is exactly accurate, since we can perform writes to previously-allocated blocks.  e.g. ZIL blocks, or healing/resilvering writes.  IMO, we're better off with something like `set vdev:allocatable=off` and `(noalloc)` or `(non-allocating)`, even though the meaning is not immediately obvious, rather than something like `set vdev:writeable=off` and `(no writes)` whose meaning is obvious but slightly wrong.